google_deployment_accounts/variables.tf

```diff
@@ -40,3 +40,9 @@ variable "github_repository" {
   type        = string
   description = "The Github repository running the deployment workflows in the format org/repository"
```

Comment:
Note that this variable still appears to be required. I was planning to make it optional in my PR: https://github.com/mozilla/terraform-modules/pull/148/files#diff-cf11d21cae1e82df3acc59244bf503b8992e1d0bf9590b2b8ad0ee5e2f5f44adR93. Making it optional makes the it possible to specify neither `github_repositories` nor `github_repository` and have the `coalescelist` fail but that seems like an acceptable and obvious error condition to module users.

Another option would to continue to require one github repository be specified this way and rename `github_repository` to e.g. `additional_github_repositories`.

Reply:
This was fixed in https://github.com/mozilla/terraform-modules/pull/152